packages/components/.storybook/main.js

```diff
@@ -0,0 +1,52 @@
+const sassFunctions = require('sass-functions');
+const sass = require('sass');
+const sassGlobImporter = require('node-sass-glob-importer');
+
+const getStyleLoaders = ({ modules }) => (
+  [
+    'style-loader',
+    {
+      loader: 'css-loader',
+      options: {
+        modules: modules,
+      }
+    },
+    {
+      loader: 'sass-loader',
+      options: {
+        implementation: sass,
+
+        sassOptions: {
+          includePaths: [
```

Comment:
same options as in rollup.config.js but not sure the effort to refactor (cfs <-> esm)